scripts/release

```diff
@@ -407,6 +407,12 @@ publish_release() {
   fi
 }
 
+publish_podspec() {
+  cd "$rootdir"
+  echo "running a pod trunk push..."
```

Comment:
Add step:
echo "Verifying Cocoapod trunk permissions"
Handle error on pod trunk me

Reply:
This is great, thanks